app/move/controllers/update/base.test.js

```diff
@@ -0,0 +1,573 @@
+const FormController = require('hmpo-form-wizard').Controller
+const { cloneDeep } = require('lodash')
+const proxyquire = require('proxyquire')
+
+const personService = require('../../../../common/services/person')
+const CreateBaseController = require('../create/base')
+const BaseProto = CreateBaseController.prototype
+
+const UpdateBaseController = proxyquire('./base', {
```

Comment:
You can probably do away with `proxyquire` here. I try to avoid using it unless absolutely necessary.

It doesn't look like this is changing any behaviour as you're loading in the same library?

Reply:
Begone proxyquire